exercises/proverb/README.md

```diff
@@ -1,16 +1,18 @@
 # Proverb
```

Comment:
As soon as exercism/problem-specifications#994 is merged, this will no longer be in sync.